pkg/apis/vsphereproviderconfig/v1alpha1/vsphereclusterproviderconfig_types.go

```diff
@@ -47,10 +47,10 @@ type VsphereClusterProviderConfig struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	VsphereUser             string `json:"vsphereUser"`
-	VspherePassword         string `json:"vspherePassword"`
+	VsphereUser             string `json:"vsphereUser,omitempty"`
```

Comment:
This looks like a patch for a previous commit and has nothing to do with ntp servers at all.  It's fine, but you need to document this in the commit.  Also, I noticed your commit has no description body.  Please add that, and comment that you also fixed some issues with a previous commit.

https://chris.beams.io/posts/git-commit/

Reply:
Done